instrumentation-appender-api-internal/src/main/java/io/opentelemetry/instrumentation/api/appender/internal/AgentLogEmitterProvider.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.appender.internal;
```

Comment:
Just confirming this makes sense in instrumentation-api and not javaagent-instrumentation-api?

Reply:
good idea, done 👍